f2/src/forms/HowDidItStartForm.js

```diff
@@ -144,90 +143,50 @@ export const HowDidItStartForm = props => {
-                        h={36}
-                        mt={2}
-                        maxLength="2"
-                      />
-                    </FormLabel>
-                  </FormControl>
-                )}
-              </Field>
-              <Field name="startMonth">
-                {props => (
-                  <FormControl>
-                    <FormLabel htmlFor="startMonth">
-                      <Trans id="whenDidItStart.startMonth" />
-                      <TextInput
-                        id="startMonth"
-                        name={props.input.name}
-                        value={props.input.value}
-                        onChange={props.input.onChange}
-                        w={70}
-                        h={36}
-                        mt={2}
-                        maxLength="2"
-                      />
-                    </FormLabel>
-                  </FormControl>
-                )}
-              </Field>
-              <Field name="startYear">
-                {props => (
-                  <FormControl>
-                    <FormLabel htmlFor="startYear">
-                      <Trans id="whenDidItStart.startYear" />
-                      <TextInput
-                        id="startYear"
-                        name={props.input.name}
-                        value={props.input.value}
-                        onChange={props.input.onChange}
-                        w={110}
-                        h={36}
-                        mt={2}
-                        maxLength="4"
-                      />
-                    </FormLabel>
-                  </FormControl>
-                )}
-              </Field>
-            </Stack>
+            <FormArrayControl
+              label={<Trans id="whenDidItStart.label" />}
+              helperText={<Trans id="whenDidItStart.labelExample" />}
+            >
+              <Stack direction="row" spacing="2">
+                <Field
+                  name="startDay"
+                  label={<Trans id="whenDidItStart.startDay" />}
+                  component={TextInput}
+                  w={70}
+                  maxLength="2"
+                />
+                <Field
+                  name="startMonth"
+                  label={<Trans id="whenDidItStart.startMonth" />}
+                  component={TextInput}
+                  w={70}
+                  maxLength="2"
+                />
+                <Field
+                  name="startYear"
+                  label={<Trans id="whenDidItStart.startYear" />}
+                  component={TextInput}
+                  w={110}
+                  maxLength="4"
+                />
+              </Stack>
+            </FormArrayControl>
 
             <FormArrayControl
               name="howManyTimes"
               label={<Trans id="howManyTimes.label" />}
             >
               {howManyTimes.map(key => {
                 return (
-                  <Box key={key}>
```

Comment:
Using a React.Fragment means we dont add another div in the rendered markup. Allows you to wrap outputs into a single react element